### DISCUSSION TITLE
DMTF's SPDM Forum is Online

### DISCUSSION THREAD

**Discussion by steven-bellock** [DMTF/libspdm#1223] [Announcements]
The DMTF SPDM Working Group has created a question and discussion forum for the SPDM specifications at https://spdmforum.com/. Folks who ask questions here about the specification will be directed to the SPDM forum.

**Comment by LeeTroy** [comment]
Is the forum still working?  I always got `ERR_TOO_MANY_REDIRECTS` error.

**Reply by steven-bellock** [reply]
Me too. I'll let them know.

**Reply by LeeTroy** [reply]
Thank you.
